core/app/c/[communitySlug]/pubs/[pubId]/components/PubValues.tsx

```diff
@@ -221,6 +219,17 @@ const PubValue = ({ value }: { value: FullProcessedPubWithForm["values"][number]
 		}
 	}
 
+	if (value.schemaName === CoreSchemaType.RichText) {
+		return (
+			<ShowMore animate={false}>
+				<div
+					className="prose prose-sm"
+					dangerouslySetInnerHTML={{ __html: value.value as string }}
+				/>
+			</ShowMore>
+		);
+	}
```

Comment:
ideally we wouldnt do manual check here, but have InputComponents maybe define some default `ReadOnly` version of themselves which can be displayed here.

Reply:
i might look into this here #1209